lib/MTT/Test/Build.pm

```diff
@@ -454,6 +454,10 @@ sub _do_build {
         $ret->{start_timestamp} = timegm(gmtime());
         $ret->{refcount} = 0;
 
+        if ($ret->{noop_installdir}) {
+            $ret->{srcdir} = $ret->{noop_installdir};
```

Comment:
what is a use-case?
why srcdir=installdir?

Reply:
This is required to pass through a path to binaries to Run section (`srcdir` is used to search for binaries for a test). I used `installdir` to highlight that it is a directory which should contain binaries, not sources.